Formula/a/act_runner.rb

```diff
@@ -0,0 +1,62 @@
+class ActRunner < Formula
+  desc "Action runner for Gitea based on Gitea's fork of act"
+  homepage "https://docs.gitea.com/usage/actions/act-runner"
+  url "https://gitea.com/gitea/act_runner/archive/v0.2.11.tar.gz"
+  sha256 "8b317700d1f3344d8664be9edb004914723a4aacc8f8b1b3719ca2260a5866b6"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    # `act_runner --version` won't work properly without this file present
+    # during the build.
+    File.write("VERSION", version)
+    # GOFLAGS is used to pass the `-trimpath` flag in `std_go_args` since the
+    # Makefile hardcodes the `go build` invocation.
+    system "make", "build", "GOFLAGS=-trimpath"
+    bin.install "act_runner"
```

Comment:
I think we just build our own,
```suggestion
    ldflags = %W[
      -s -w
      -X gitea.com/gitea/act_runner/internal/pkg/ver.version=v#{version}
    ]
    system "go", "build", *std_go_args(ldflags:)
    generate_completions_from_executable(bin/"act_runner", "completion")
```